tools/percy/snapshots.js

```diff
@@ -1,111 +1,114 @@
+    name: 'About page',
     navPage: true,
   },
   // Remove this until https://github.com/GoogleChrome/web.dev/issues/4790 is fixed.
   // {
   //   url: '/codelab-avoid-invisible-text/',
-  //   title: 'Codelab page',
+  //   name: 'Codelab page',
   // },
   {
     url: '/handbook/web-dev-components/',
-    title: 'Components page',
+    name: 'Components page',
   },
   {
     url: '/authors/',
-    title: 'Authors page',
+    name: 'Authors page',
   },
   {
     url: '/authors/mgechev/',
-    title: 'Author page',
+    name: 'Author page',
   },
   {
     url: '/tags/',
-    title: 'Tags page',
+    name: 'Tags page',
   },
   {
     url: '/shows/',
-    title: 'Shows page',
+    name: 'Shows page',
   },
   {
     url: '/podcasts/',
-    title: 'Podcasts page',
+    name: 'Podcasts page',
   },
   {
     url: '/newsletter/',
-    title: 'Newsletter page',
+    name: 'Newsletter page',
   },
   {
     url: '/live/',
-    title: 'Live page',
+    name: 'Live page',
     navPage: true,
   },
 ];
 
-async function waitForNavUpdate(page, browser) {
+async function waitForNavUpdate(pageToTest, page) {
   try {
-    page.navPage &&
-      (await browser.waitFor(
+    pageToTest.navPage &&
+      (await page.waitFor(
         (url) => {
           const activeLink = document.querySelector('a[active=""]');
 
           return activeLink && activeLink.getAttribute('href') === url;
         },
         {polling: 15, timeout: 5000},
-        page.url,
+        pageToTest.url,
       ));
   } catch (err) {
-    console.log(`Couldn't find active link for ${page.url}`);
+    console.log(`Couldn't find active link for ${pageToTest.url}`);
   }
 }
 
 // A script to navigate our app and take snapshots with Percy.
-PercyScript.run(
-  async (browser, percySnapshot) => {
-    // set the viewport to a desktop size
-    await browser.setViewport({height: 1024, width: 1280});
+(async () => {
+  const browser = await puppeteer.launch({
+    args: ['--no-sandbox', '--disable-setuid-sandbox'],
+  });
+  await browser.setViewport({height: 1024, width: 1280});
 
-    for (const page of pagesToTest) {
-      const url = new URL(page.url, 'http://localhost:8080').href;
-      await browser.goto(url, {waitUntil: 'networkidle0'});
-      await browser.evaluate(scrollToBottom);
-      // Wait for the SPA to update the active link in the top nav.
-      await waitForNavUpdate(page, browser);
-      await percySnapshot(`${page.title}`);
-    }
-  },
-  // These flags remove Percy's single-process flag. Without these, any page
-  // with an iframe will crash puppeteer (and percy).
-  {args: ['--no-sandbox', '--disable-setuid-sandbox']},
-);
+  for (const pageToTest of pagesToTest) {
+    const page = await browser.newPage();
+    const url = new URL(pageToTest.url, 'http://localhost:8080').href;
+    await page.goto(url, {waitUntil: 'networkidle0'});
+    await page.evaluate(scrollToBottom);
+    // Wait for the SPA to update the active link in the top nav.
```

Comment:
We don't do spa atm. Do we still need this?

Reply:
I don't think we do, I'll remove it, and if we end up needing it we could always add it back...